api/ooniapi/citizenlab.py

```diff
@@ -794,7 +794,6 @@ def post_propose_changes() -> Response:
 
 
 @cz_blueprint.route("/api/_/url-priorities/list", methods=["GET"])
```

Comment:
Should we expose it in the public API instead?

Reply:
I think we should keep it here since it's currently only used internally and I don't think we want to make any guarantees for future backward compatibility of this API.